include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
I get this warning many times on vs2015, if it doesnt appear on any other version of vs, its probably vs2015 specific. Do you have the goal to do warning free builds on any supported compiler?